app/views/directives/edit-environment-from.html

```diff
@@ -67,6 +67,11 @@
           role="button"
           aria-label="Delete row"
           ng-click="$ctrl.deleteEntry($index, 1)"></a>
+        <a
+          ng-if="!$ctrl.editEnvironmentFromForm.$dirty"
```

Comment:
This won't hide the link if you have variables form above is dirty.

Reply:
This logic should be getting removed since we're headed in the recommended direction of displaying the "details" modal link as soon as a value is selected for the list.